src/executor/load_executor.py

```diff
@@ -0,0 +1,50 @@
+# coding=utf-8
+# Copyright 2018-2020 EVA
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from src.planner.load_data_plan import LoadDataPlan
+from src.executor.abstract_executor import AbstractExecutor
+from src.storage.dataframe import append_rows
+from src.readers.opencv_reader import OpenCVReader
+
+
+class LoadDataExecutor(AbstractExecutor):
+
+    def __init__(self, node: LoadDataPlan):
+        super().__init__(node)
+
+    def validate(self):
+        pass
+
+    def exec(self):
+        """
+        Read the input video using opencv and persist data
+        using storage engine
+        """
+        # Arbitrary batch size
```

Comment:
We could move the `batch_size` to the configuration file?

Reply:
Done